aod-discord-bot.js

```diff
@@ -4251,15 +4252,19 @@ client.on('guildAuditLogEntryCreate', async function(auditLogEntry, guild) {
 					}
 				}
 			});
+
+			if (actionDescription === undefined) {
+				console.log(`Unsupported MemberUpdate on ${auditLogEntry.target.username} by ${auditLogEntry.executor.username}: ${JSON.stringify(auditLogEntry.changes)}`);
+				return;
+			}
+
 			break;
 		default:
 			return;
 	}
 
 	let reason = auditLogEntry.reason ?? 'No reason provided';
-	let executor = await client.users.fetch(auditLogEntry.executorId);
-	let target = await client.users.fetch(auditLogEntry.targetId);
-	await global.sendGlobalNotification(guild, `${target} has been ${actionDescription} by ${executor} for: ${reason}`);
+	await global.sendGlobalNotification(guild, `${auditLogEntry.target} has been ${actionDescription} by ${auditLogEntry.executor} for: ${reason}`);
```

Comment:
I actually did the same thing when I updated originally, but this gives the discord user not the guild member. Please double check that the mention is proper.

Reply:
It's passing the numeric user ID in the message body so we should be fine either way unless you ran into an edge case that I missed. 

`<@939640076185518090> has been removed from timeout by <@66275451470880768> for: No reason provided` 

![image](https://github.com/user-attachments/assets/dff5f6f8-86ce-459a-bd33-91c06528a177)